about/index.html

```diff
@@ -1,12 +1,30 @@
 <!DOCTYPE html>
```

Comment:
Make your HTML files prettier. Remove extra spaces and write each tag with a new line.
```suggestion
<p>
     <span>Some text<span>
</p>
```